src/common/personinfo.js

```diff
@@ -167,8 +167,34 @@ let personinfo = function (tour, aConfig) {
   });
 };
 
+
+const PERSISTENT_LOCALSTORAGE_KEY="localStorageUsed";
+let tryLocalStorage = function (store=localStorage) {
```

Comment:
Out of curiosity, why `let foo = function ()` rather than just `function foo()` ?